CONTRIBUTING.md

```diff
@@ -318,40 +345,38 @@ You will also need several other mkdocs plugins, like `branchcustomization` and
 To install all of this make sure you have a recent version of Python on your computer.
 The [DataLad Handbook](http://handbook.datalad.org/en/latest/intro/installation.html#python-3-all-operating-systems) provides helpful instructions for setting up Python.
 
-An easy way to install the correct version of mkdocs and all the other required extensions
-is to use the `requirements.txt` file contained in this repository,
-by using the following command:
+In general, we strongly recommend that you install all dependencies in an isolated Python environment.
+For example using `conda`, as described in the section on Python in the DataLad Handbook, linked above.
 
 ```bash
-pip install -r requirements.txt
+conda create --name bids-spec
+conda activate bids-spec
 ```
 
-However this will also install some other packages you might not want to have (like `numpy`).
-So if you only want to install what you need to build the specification,
-use the following command:
+Or alternatively using `venv`, as described in this [Real Python tutorial](https://realpython.com/python-virtual-environments-a-primer/).
+
+A short version of the commands needed to create and activate your `venv` virtual environment would look like:
 
 ```bash
-pip install \
- mkdocs \
- mkdocs-material \
- pymdown-extensions \
- mkdocs-branchcustomization-plugin \
- mkdocs-macros-plugin \
- tabulate
+python3 -m venv env
+source env/bin/activate
 ```
 
-#### 2. Download the BIDS specification [repository](https://github.com/bids-standard/bids-specification/tree/master) onto your computer
+Note however, that this will create a local directory called `env` within the bids-specification directory.
+Make sure not to commit that directory (your environment) when adding  and committing your changes via `git`
```

Comment:
```suggestion
Note that this will create a local directory called `env` within the bids-specification directory but that its content will not be tracked by GIT because it is listed in the `.gitignore` file.
```

Reply:
ahh :+1: thanks!